var/spack/repos/builtin/packages/memsurfer/package.py

```diff
@@ -12,32 +12,31 @@ class Memsurfer(PythonPackage):
        wide variety of large-scale molecular simulations."""
 
     homepage = "https://github.com/LLNL/MemSurfer"
-    git      = "git@github.com:LLNL/MemSurfer.git"
-    # url      = "https://github.com/LLNL/MemSurfer/archive/1.0.tar.gz"
+    git = "git@github.com:LLNL/MemSurfer.git"
 
-    # version('1.0', sha256='06e06eba88754b0c073f1c770981f7bdd501082986e4fbe28399be23b50138de')
-    version('1.0', tag='v1.0', submodules=True)
-    version('master', branch='master', submodules=True)
-    # version('test', branch='ppoisson', submodules=True)
+    version('1.0',     tag='v1.0',       submodules=True)
+    version('master',  branch='master',  submodules=True)
+    version('develop', branch='develop', submodules=True)
 
-    variant('vtkmesa', default=False, description='Enable OSMesa support for VTK')
+    variant('osmesa', default=False,
+            description='Enable OSMesa support (for VTK)')
+
+    extends('python@3.7:')
```

Comment:
Python 2 is no longer supported in v1.0?

Reply:
I have moved to Python3. I do not think p2 is supported anymore...

Reply:
I think this should be type=build/run, right?

Reply:
you mean the `extends` call? I didn't know it also had types.. the default should be both because the package builds and runs.

Reply:
It actually doesn't have types, it just assigns `build/link` by default. You can add:
```python
depends_on('python@3.7:', type=('build', 'run'))
```
below it. And then it will combine those to make it build/link/run.

Reply:
done